README.md

```diff
@@ -1,31 +1,30 @@
 
 [![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg)](http://bioconda.github.io/)
 [![Docker](https://img.shields.io/docker/automated/nfcore/rnafusion.svg)](https://hub.docker.com/r/nfcore/rnafusion)
 
-**nfcore/rnafusion** uses RNA-seq data to detect fusions genes.
-
-The workflow processes RNA-sequencing data from FastQ files. It runs quality control on the raw data ([FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)), detects fusion genes ([STAR-Fusion](https://github.com/STAR-Fusion/STAR-Fusion), [Fusioncatcher](https://github.com/ndaniel/fusioncatcher), [Ericscript](https://sites.google.com/site/bioericscript/), [Pizzly](https://github.com/pmelsted/pizzly), [Squid](https://github.com/Kingsford-Group/squid)), gathers information ([FusionGDB](https://ccsm.uth.edu/FusionGDB/index.html), [Mitelman](https://cgap.nci.nih.gov/Chromosomes/Mitelman), [COSMIC](https://cancer.sanger.ac.uk/cosmic/fusion)), visualizes the fusions ([FusionInspector](https://github.com/FusionInspector/FusionInspector)), performs quality-control on the results ([MultiQC](http://multiqc.info)) and finally generates custom summary report witch scored fusions ([fusion-report](https://github.com/matq007/fusion-report)).
-
-> Live **demo** output **[here](https://matq007.github.io/fusion-report/example/)**.
+## Introduction
 
-The pipeline works with both single-end and paired-end data, though not all fusion detection tools work with single-end data (Ericscript, Pizzly, Squid and FusionInspector).
-
-The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker / singularity containers making installation trivial and results highly reproducible.
+The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker containers making installation trivial and results highly reproducible.
 
 | Tool            | Single-end reads | CPU (recommended) | RAM (recommended) |
 | --------------- |:----------------:|:-----------------:|:-----------------:|
-| [Star-Fusion](https://github.com/STAR-Fusion/STAR-Fusion/wiki) | Yes              | >=16 cores        | ~30GB             |
-| [FusionCatcher](https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md) | Yes              | >=16 cores        | ~64GB             |
+| [Arriba](https://github.com/suhrig/arriba) | **No**           | >=16 cores        | ~30GB             |
 | [EricScript](https://sites.google.com/site/bioericscript/getting-started) | **No**           | >=16 cores        | ~30GB             |
+| [FusionCatcher](https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md) | Yes              | >=16 cores        | ~64GB             |
```

Comment:
```suggestion
| [FusionCatcher](https://github.com/ndaniel/fusioncatcher) | Yes              | >=16 cores        | ~64GB             |
```
What do you think about a link just to the repo?

Reply:
Good point 👍